src/main/webapp/resources/js/components/Buttons/AddMemberButton.jsx

```diff
@@ -19,9 +19,9 @@ const { Text } = Typography;
 export function AddMemberButton({
   label,
   modalTitle,
-  addMemberFn = () => {},
-  getAvailableMembersFn = () => {},
-  addMemberSuccessFn = () => {},
+  addMemberFn = () => undefined,
```

Comment:
Great question. We should ask the author.

Reply:
Yes, I know I was the author of this.  Since the time I wrote it and now reviewing this code that you have updated I realized that it should have been done differently.  All I would like you to do please, is to remove the empty function calls so that if this component is used in the future and those functions are not passed in, an error will be thrown when they are called.

Reply:
Please see 4cf05f5154b16538c644bccc5e8775ef070cda1e.

Reply:
Thank you.